src/Control/Session.php

```diff
@@ -497,9 +493,7 @@ public function getAll()
 
     /**
      * Set user agent key
-     *
-     * @param HTTPRequest $request
-     */
+*/
```

Comment:
Missing whitespace here (unless it's GitHub messing up the preview)

Reply:
done